vm-systemd/qubes-qrexec-agent.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Qubes remote exec agent
-After=qubes-dvm.service
+After=qubes-dvm.service xendriverdomain.service
```

Comment:
> So all the code from marmarek/qubes-vmm-xen@8639b69 could be avoided?

Not really, it is still needed, but having native xendriverdomain.service will make it easier.